server/s3_iem.h

```diff
@@ -66,37 +66,49 @@ extern S3Option* g_option_instance;
 // IEM helper macros
 // S3 Server IEM Event codes, description string & json data format
 #define S3_IEM_AUTH_CONN_FAIL "0030010001"
-#define S3_IEM_AUTH_CONN_FAIL_STR "Auth server connection failed"
+#define S3_IEM_AUTH_CONN_FAIL_STR                                         \
+  "S3 server cannot connect to auth server. For more information, refer " \
+  "troubleshooting guide."
 #define S3_IEM_AUTH_CONN_FAIL_JSON ""
 
 #define S3_IEM_CHUNK_PARSING_FAIL "0030020001"
-#define S3_IEM_CHUNK_PARSING_FAIL_STR "Chunk parsing failed"
+#define S3_IEM_CHUNK_PARSING_FAIL_STR \
+  "Invalid chunk payload. For more information, refer troubleshooting guide."
 #define S3_IEM_CHUNK_PARSING_FAIL_JSON ""
 
 #define S3_IEM_MOTR_CONN_FAIL "0030030001"
-#define S3_IEM_MOTR_CONN_FAIL_STR "Motr connection failed"
+#define S3_IEM_MOTR_CONN_FAIL_STR                                          \
+  "Clovis connection failed. For more information, refer troubleshooting " \
```

Comment:
please remove Clovis word and use Motr client may be ?

Reply:
Renamed clovis to motr.

Reply:
It should be "... refer Troubleshooting Guide."
Add full stop (.) after every error message.